slave/build.py

```diff
@@ -251,6 +279,8 @@ def getBuilder(config, path):
         return WebkitBuilder(config, path)
     if os.path.exists(os.path.join(path, "v8", "LICENSE.v8")):
         return V8Builder(config, path)
+    if os.path.exists(os.path.join(path, "mach")):
```

Comment:
Can we be a bit more specific? mozilla-inbound also has "mach". And I don't want this list to depend on the order of execution. Do we have a structure/file that is very specific for servo?